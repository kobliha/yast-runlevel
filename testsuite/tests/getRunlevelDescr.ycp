// test for getRunlevelDescr
{
    include "testsuite.ycp";
    include "runlevel/ui.ycp";

    DUMP (getRunlevelDescr ("0"));
    DUMP (getRunlevelDescr ("1"));
    DUMP (getRunlevelDescr ("2"));
    DUMP (getRunlevelDescr ("3"));
    DUMP (getRunlevelDescr ("4"));
    DUMP (getRunlevelDescr ("5"));
    DUMP (getRunlevelDescr ("6"));
    DUMP (getRunlevelDescr ("7"));    
}
