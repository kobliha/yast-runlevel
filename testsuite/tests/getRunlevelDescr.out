Dump	Unchanged
Dump	0: System halt
Dump	1: Single user mode
Dump	2: Local multiuser without remote network
Dump	3: Full multiuser with network
Dump	4: User defined
Dump	5: Full multiuser with network and xdm
Dump	6: System reboot
Dump	7
